public/resource/chains.json

```diff
@@ -1,10 +1,10 @@
 [
   {
-    "name": "Testnet 4",
-    "chainId": "test4",
-    "apiUrl": "https://test4.api.onbloc.xyz/v1",
-    "rpcUrl": "https://rpc.test4.gno.land:443",
-    "indexerUrl": "https://test4.indexer.onbloc.xyz"
+    "name": "Testnet 5",
+    "chainId": "test5",
+    "apiUrl": "https://test5.api.onbloc.xyz/v1",
+    "rpcUrl": "https://rpc.test5.gno.land:443",
```

Comment:
Let's use `https://test5.onbloc.xyz/` for now.

Reply:
I updated this PR.(https://github.com/onbloc/gnoscan/pull/121/commits/fd704f65ef58dc7c040c14e2b02f1127a2ece338)